### PR TITLE
added multiple queuedwithdrawals function

### PR DESCRIPTION
**Pull request by Sidu28**
Adding ability to prove multiple queuedWithdrawals in a single function call

### CODE REVIEW

File: src/contracts/core/DelegationManager.sol [hunk 1]
```diff
@@ -264,6 +264,34 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
         });
     }
 
+    function queueWithdrawals(
+         IStrategy[][] calldata strategies,
```
Comment (gpsanant) [inline src/contracts/core/DelegationManager.sol:268]: i think we should create a `QueuedWithdrawaParam` struct and the param here should be `QueuedWithdrawaParam[]`

File: src/contracts/core/DelegationManager.sol [hunk 2]
```diff
@@ -264,6 +264,34 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
...
+        uint256[][] calldata shares,
+        address[] calldata withdrawers
+    ) external onlyWhenNotPaused(PAUSED_ENTER_WITHDRAWAL_QUEUE) returns (bytes32[] memory) {
+        require(strategies.length == shares.length && shares.length == withdrawers.length, "DelegationManager.queueWithdrawals: input length mismatch");
+        bytes32[] memory withdrawalRoots = new bytes32[](strategies.length);
```
Comment (gpsanant) [inline src/contracts/core/DelegationManager.sol:273]: also, please delete the other queueWithdrawal function

**Review by ChaoticWalrus — COMMENTED** [review]
Seems reasonable, would really like to see some test coverage added.

Should we leave a singleton method? I guess it's fine just having the array-ified version though.

Have you already coordinated with others on this interface change?

**Review by ChaoticWalrus — APPROVED** [review]
I am approving but I'd say wait for @gpsanant to confirm prior to merging

**Comment by Sidu28** [comment]
> Seems reasonable, would really like to see some test coverage added.
> 
> Should we leave a singleton method? I guess it's fine just having the array-ified version though.
> 
> Have you already coordinated with others on this interface change?

Gautham asked for this to replace the singleton method

UI is ok with the interfact change since apparently no balance updates for M2

**Pull request merged**